recipe/meta.yaml

```diff
@@ -89,6 +89,8 @@ requirements:
     - vc 14  # [win and (py35 or py36)]
 
 test:
+  requires:
+    - libstdcxx-ng  # [linux]
```

Comment:
Add this to run requirements

Reply:
I tested the new `{{ compiler('cxx') }}` but that requires more recipes changes that I don't think are worth doing at the moment. Also, I faced an issue with everything that uses `--with-geos=$PREFIX/bin/an-executable` where the executable could not be found.

Since in our old model the `libstdcxx` is expected to be in the user machine I won't add it to the run requirements to avoid surprises. I'd rather focus on rebuild our stack with the new compilers and merging our recipes with the AnacondaAggregate to solve this mess.

Reply:
> Since in our old model the libstdcxx is expected to be in the user machine I won't add it to the run requirements to avoid surprises. 

Sure. Note that you'll have to add it for all packages that have libgdal as a dependency because the docker image doesn't have it.

Reply:
> Sure. Note that you'll have to add it for all packages that have libgdal as a dependency because the docker image doesn't have it.